src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name.startswith("D")]
 print(a)
```

Comment:
Good use of python methods rather than directly referencing the string position as others have done.